src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -411,7 +411,7 @@
   &.pf-m-primary,
   &.pf-m-secondary {
     // Reduce the padding before/after the border unless it's an icon/check-only element
-    .#{$menu-toggle}__button:not(:has(.#{$menu-toggle}__toggle-icon:only-child)),
+    .#{$menu-toggle}__button:not(:has(.#{$menu-toggle}__controls:only-child)),
```

Comment:
This was just targeting the wrong element. Needs to look for `__controls` as an only child (meaning no `__text` sibling). It's effectively the same style as this one, but in `:not()` to target the inverse https://github.com/patternfly/patternfly/blob/689a71891bac6823c3dc2bd1406ea713f7037e3d/src/patternfly/components/MenuToggle/menu-toggle.scss#L486